src/lib/actions/inspections.ts

```diff
@@ -0,0 +1,133 @@
+import { tenantPrisma } from '$lib/prisma';
+
+/*
+ *	Get all inspections
+ */
+export const fetchInspections = async ({ tenantId }: { tenantId: string }) => {
```

Comment:
Este es un problema grave. Estás creando una instancia de prisma client cada vez que haces una consulta. En realidad esta instancia debe crearse donde se selecciona el tenantId  moverla al lado del mismo para cada consulta.
Mira el mensaje de error:
"warn(prisma-client) This is the 10th instance of Prisma Client being started. Make sure this is intentional."

Reply:
No he visto esa advertencia pero se que se trata de un error en el archivo prisma.ts donde se inicia la instancia.